### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by sagieramos**
- Added HTML
- Added Stylesheet
- Added README

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (joel-kalema) [inline index.html:3]: #### Great job working on setting up a 'Hello Microverse' project 👍. however, you need to fix some parts:

- [ ] Currently, your project doesn't have a descriptive pull request. Kindly make sure you added some information about what you have done in this project to make it more descriptive 😃

Capture

- [ ] Currently, talking about linter, it helps us clean and standardize our code regarding best practices and DRY method, Kindly make sure you fixed then 👍 

Capture

- [ ] Talking about README

 - Kindly make sure you deleted the [Microverse logo](https://github.com/sagieramos/Hello-world/tree/feature#:~:text=Microverse%20README%20Template) or you may replace with your own if you have one 😄

Capture

  - Kindly make sure you customized the sections below, you may show how we can `Setup`, `Install`, and `Usage` your project, also, you may remove the `Run tests` and `Deployment` sections. Aloso make sure you added a `Description` for this project. Kindly fix it to make your README more descriptive 👍

Capture

  -  Make sure you added all information about your social media for `👤 Author1` and remove `👤 Author2`.  Kindly, fix it 👍  

Capture

 - Make sure your MIT license is working, you may add the like to your `MIT` file. It'll be some thing like `This project is [MIT](./MIT.md) licensed.` . Kindly fix it 👍 

Capture

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @sagieramos 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (sagieramos) [reply index.html:3]: ### Set up a "Hello Microverse" project
**Issues fixed**
- The README was updated. 
- A new MIT license was added.

**Comment by sagieramos** [comment]
README.md was upadated accordingly

File: README.md [hunk 2]
```diff
@@ -60,69 +26,40 @@ After you're finished please remove all the comments and instructions!
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+- [Live Demo Link](./index.html)
```
Comment (divinecharlotte) [inline README.md:55]: - [ ] As requested by the previous reviewer kindly consider removing this section as you are not going to deploy this project and doing so will make your readme more professional

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @sagieramos     , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- linters are passing 👍





 

## Required Changes ♻️

- [ ] Kindly as requested by the previous reviewer consider adding a PR description to your pr to give more information on what you did in this branch because currently its empty

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aamirkhan2478 — APPROVED** [review]
Hi @sagieramos , 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly
✅ No linter error
✅ Good commit messages
✅ Good descriptive PR
✅ Well done you did all the changes give by the previous reviewers :clap: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

[OPTIONAL] This is optional because the previous reviewers missed this requirement please edit your PR title it's not descriptive replace `Feature` with `Hello Microverse` and please don't skip this requirement :slightly_smiling_face: 

[OPTIONAL] This is optional because the previous reviewers missed this requirement please use a descriptive branch name. The branch name should be any action or feature like `add-heading` so kindly next time when you work on another project please keep in mind these requirements :+1: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @sagieramos 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- [ ] As suggested by the previous reviewer please ensure that the PR title is descriptive. This gives the PR visitors a clear understanding of what the pull request is all about. In addition, please ensure to make the branch name more descriptive to avoid project invalidations 
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**